experiments/experiment.go

```diff
@@ -416,6 +415,19 @@ func (ec *experimentContext) reconcileAnalysisRun(analysis v1alpha1.ExperimentAn
 				eventType = corev1.EventTypeWarning
 			}
 			ec.recorder.Eventf(ec.ex, record.EventOptions{EventType: eventType, EventReason: "AnalysisRun" + string(newStatus.Phase)}, msg)
```

Comment:
I go back on forth on if we should still emit this event when it's emitted on the Rollout resource. It's pretty low volume event but the duplication seems funny.

Trying to think think of what makes the most logical sense.

Reply:
It would be breaking to change this so let's keep it

Reply:
Yeah, and I think its fine, since in case someone has rolled their own notif handlers, like you say, we don't break it. we are already duping it (there is one emitted that belongsa to the analysis run itself).